### DISCUSSION TITLE
How to convert selected content of markdown file to slides/ppt?

### DISCUSSION THREAD

**Discussion by ssharma397** [marp-team/marp#324] [Q&A]
I have a markdown file that contains the detailed explaination about some of the topics. I want to use few of those contents (not all) and convert it into ppt file.  Is there any way this can be done? Please guide.

**Comment by yhatt** [comment]
Currently Marp has no ability to convert only specific pages. You have to make a new Markdown file with partial contents by pasting picked pages.

You can create a new issue to request the feature into [Marp CLI](https://github.com/marp-team/marp-cli) if you think that an option like `--pages 1,3-5` is helpful.
